hydro_deploy/core/src/localhost/mod.rs

```diff
@@ -193,6 +195,51 @@ impl LaunchedHost for LaunchedLocalhost {
             (None, command)
         };
 
+        // from cargo
+        let dylib_path_var = if cfg!(windows) {
+            "PATH"
+        } else if cfg!(target_os = "macos") {
+            "DYLD_FALLBACK_LIBRARY_PATH"
+        } else if cfg!(target_os = "aix") {
+            "LIBPATH"
+        } else {
+            "LD_LIBRARY_PATH"
+        };
```

Comment:
Link to the cargo source in a comment if you think this might ever need to be touched?

Reply:
Else case is linux?